circuit/src/get_account.nr

```diff
@@ -6,20 +6,19 @@ struct AccountWithProof {
     balance: Field,
     codeHash: [u8; 32],
     nonce: Field,
-    state_root: [u8; proof::KEY_LENGTH],
     key: [u8; 20],
     value: [u8; proof::MAX_ACCOUNT_STATE_LENGTH],
     proof: [u8; PROOF_LEN],
     depth: Field,
 }
 
 #[oracle(get_account)]
-unconstrained fn get_account_oracle(_block_no: Field, _address: [u8; 20]) -> AccountWithProof {}
+unconstrained fn get_account_oracle(_block_no: Field, _address: [u8; 20], _state_root: [u8; proof::KEY_LENGTH]) -> AccountWithProof {}
```

Comment:
No need for `_state_root` in actual oracle `get_account_oracle`.
It is only used in `get_account`. Also see comment below.

Reply:
done